cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -349,8 +346,10 @@ Stripe::updateHeaderFooter()
     for (int j = 0; j < 2; j++) {
       char *meta_t = (char *)ats_memalign(ats_pagesize(), this->vol_dirlen());
```

Comment:
Didn't we take the leading `vol_` off these methods?

Reply:
that was  in the ATS core